resources/helm/v2.1/gateways/istio-ingress/templates/serviceaccount.yaml

```diff
@@ -8,8 +8,8 @@ imagePullSecrets:
 {{- end }}
 {{- end }}
 metadata:
-  name: {{ $gateway.name | default "istio-ingressgateway" }}-service-account
-  namespace: {{ $gateway.namespace | default .Release.Namespace }}
+  name: {{ $gateway.name }}-service-account
+  namespace: {{ .Release.Namespace }}
```

Comment:
This needs to be reverted, or it will break installation of gateways in other namespaces.